ml-pipeline/ml-pipeline/pipeline-apiserver.libsonnet

```diff
@@ -205,66 +202,9 @@
       },
       rules: [
         {
-          apiGroups: [""],
-          resources: [
-            "pods",
-            "pods/exec",
-          ],
-          verbs: [
-            "create",
-            "get",
-            "list",
-            "watch",
-            "update",
-            "patch",
-          ],
-        },
-        {
-          apiGroups: [""],
-          resources: [
-            "secrets",
-          ],
-          verbs: [
-            "get",
-          ],
-        },
-        {
-          apiGroups: [""],
-          resources: [
-            "configmaps",
-          ],
-          verbs: [
-            "get",
-            "watch",
-            "list",
-          ],
-        },
-        {
-          apiGroups: [
-            "",
-          ],
-          resources: [
-            "persistentvolumeclaims",
-          ],
-          verbs: [
-            "create",
-            "delete",
-          ],
-        },
-        {
-          apiGroups: [
-            "argoproj.io",
-          ],
-          resources: [
-            "workflows",
-          ],
-          verbs: [
-            "get",
-            "list",
-            "watch",
-            "update",
-            "patch",
-          ],
+          apiGroups: ["*"],
```

Comment:
Could you please identify the minimum amount of permissions needed and use that?

Reply:
done